src: added support for collections

dusansimic opened the pull request:
This commit suggests adding support for getting
photos from collections.

Closes https://github.com/unsplash/unsplash-source-js/issues/6

dusansimic committed:
src: added support for collections

This commit suggests adding support for getting
photos from collections.

Ref: https://github.com/unsplash/unsplash-source-js/issues/6

lukechesser commented:
@dusansimic nice. Do you think you could [add some tests here](https://github.com/unsplash/unsplash-source-js/blob/master/tests/urls.js) covering the collections?

dusansimic committed:
src: adding tests for collections

This commit suggests adding tests tests for
collections. It also suggests the fix for previous
commit.

dusansimic commented:
Just wrote the tests.
Also added a small fix. Typo while copying the code.

lukechesser commented:
Looks great thanks @dusansimic 👍
